features/test.feature

```diff
@@ -14,7 +14,7 @@ Feature: Test the Instant Articles generators
 
     Scenario: It will have an Gallery widget
         Then the response should contain "<h1>350x150</h1>"
-        And  the response should contain "<img src=\"http://shortlist.dev/app/uploads/2017/09/350x150"
+        And  the response should contain "<img src=\"http://shortlist.dev/app/uploads/2017/10/350x150"
```

Comment:
😂 uh oh. Is this on your todo list to make this dynamic so the tests don't fail next month?

Reply:
I am aware of this one. I still need to find a way to do this... because on every new test run the dates will change